ts/olive_tr.ts

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="tr_TR">
 <context>
-    <name>AboutDialog</name>
+    <name>Olive Hakkında</name>
```

Comment:
I don't think that the context is supposed to be translated. Linguist does not allow you to do that as far as I can tell.

```suggestion
    <name>AboutDialog</name>
```